app/main/helpers/sum_counts.py

```diff
@@ -9,7 +9,7 @@ def _sum_counts(stats, filter_by=None, sum_by='count'):
     return sum(
         statistic[sum_by] for statistic in stats
         if not filter_by or all(
-            statistic.get(key) == value
+            statistic.get(key) in value if isinstance(value, list) else statistic.get(key) == value
```

Comment:
I know this is probably "Pythonic", but it might be clearer to split this logic up a bit if possible? It's making my brain hurt.